rust/helm/hyperlane-agent/values.yaml

```diff
@@ -28,11 +28,9 @@ storage:
 # Note: For successful operation, one _must_ pass signer keys as
 #       they are not baked into the image for security reasons.
 hyperlane:
-  # Sets the config folder to use
+  # Defines what environment is being run
```

Comment:
What is `runEnv` used for now that we're loading all configs?

Reply:
It is used by the k8s config to define labels and the like. Removing it would be more difficult than it is worth I think since for now our deployments are still mainnet/testnet labeled and configured. Literally with different k8s clusters.

Reply:
Agreed, as soon as this touches our cloud setup it's not worth it, that stuff is pretty entrenched